romana/client/client.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
Also we use client.go (in different package though) for rest client code. Could be confusing?